docs/source/tutorials/07_using_model_hub_nodes.rst

```diff
@@ -0,0 +1,334 @@
+:greenbox:`q` to end early.
+
+
+Object Detection
+----------------
+
+This example also demonstrates how to blur computer related objects but with Hugging
+Face Hub's object detection models instead.
+
+Supported Models
+^^^^^^^^^^^^^^^^
+
+Use the following command to get a list of supported object detection models:
+
+   .. admonition:: Terminal Session
+   
+      | \ :blue:`[~user/project_dir]` \ > \ :green:`peekingduck model-hub huggingface models -\-task 'object detection'` \
+      | Supported Hugging Face \`object_detection\` models:
+      | facebook/detr-resnet-50
+      | facebook/detr-resnet-50-dc5
+      | hustvl/yolos-base
+      | hustvl/yolos-small
+      |
+      | <omitted>
+
+Class Labels
+^^^^^^^^^^^^
+
+Similar to the instnace segmentation models, the object detection models are also
+trained on a variety of datasets, the same ``detect-ids`` command can be used to
+retrieve the file containing the model's class labels:
+
+   .. admonition:: Terminal Session
+   
+      | \ :blue:`[~user/project_dir]` \ > \ :green:`peekingduck model-hub huggingface detect-ids  -\-model_type 'hustvl/yolos-small'` \
+      | The detect ID-to-label mapping for \`facebook/hustvl/yolos-small\` can be found at https://huggingface.co/hustvl/yolos-small/blob/main/config.json under the \`id2label\` key.
+
+Pipeline
+^^^^^^^^
+
+The following ``pipeline_config.yml`` can be used to blur computer related objects:
+
+   .. code-block:: yaml
+      :linenos:
+   
+      nodes:
+      - input.visual:
+          source: /path/to/cat_and_computer.mp4
+      - model.huggingface_hub:
+          task: object_detection 
+          model_type: hustvl/yolos-small
+          detect: ["laptop", "keyboard", "mouse"]
+      - draw.blur_bbox
+      - output.screen
+
+Here is a step-by-step explanation of what has been done:
+
+   | Line 2 :mod:`input.visual` tells PeekingDuck to load ``cat_and_computer.mp4``.
+   | Line 4 :mod:`model.huggingface_hub` is set to perform the ``object_detection``
+   |        task and load the ``hustvl/yolos-small`` model. We also
+   |        set the model to detect only laptops, keyboards, and mice.
+   | Line 8 :mod:`draw.blur_bbox` is used to apply a blur effect on the bounding
+   |        boxes detected by the model.
+
+Run the above with the command :greenbox:`peekingduck run`. 
+
+   .. figure:: /assets/tutorials/ss_huggingface_object_detection.png
+      :width: 416
+      :alt: Cat and Computer Screenshot with Computer Related Objects Blurred
+   
+      Cat and Computer Screenshot with Computer Related Objects Blurred
+
+
+You should see a display of the ``cat_and_computer.mp4`` with the computer related
+objects blurred. The 30-second video will auto-close at the end, or you can press
+:greenbox:`q` to end early.
+
+
+
+MediaPipe Solutions
+===================
+
+The :mod:`model.mediapipe_hub` node supports MediaPipe solutions which perform the
+following computer vision tasks:
+
+   #. Object detection
+   #. Pose estimation
```

Comment:
We would want to be specific here and highlight that only Face Detection is supported currently, not the generic C++ Object Detection model (unless there's some way to get `bazel` to compile the tflite model as per MediaPipe's instructions).

Reply:
Added a note immediately below to say that only face detection is available as the generic object detection model is not available in Python.

I opted to keep the "Object detection" in the list of tasks instead of changing to the "Face detection" as I feel it is more consistent with how the CLI commands and config values are used.